### ISSUE TITLE
sbt install does not work for windows

### ISSUE THREAD

**Issue by JonathanStarup** [effekt-lang/effekt#143]
`sbt install` did not work for me on windows and I had to use `npm.cmd` instead of `npm` to fix it ([SO discussion](https://stackoverflow.com/questions/22708255/cannot-run-program-npm-in-directory)).

This is not a fix since it breaks Unix but the workaround is here `build.sbt:136`
```sbt
install := {
      assembleBinary.value
      Process("npm.cmd pack").!!
      Process(s"npm.cmd install -g effekt-${effektVersion}.tgz").!!
    },
```

**Comment by b-studios** [comment]
For easier reference, here is the original version:

https://github.com/effekt-lang/effekt/blob/75016e7025448de5163f3fed1e977daf880ea170/build.sbt#L136-L140

**Comment by b-studios** [comment]
Here is an SO post on detecting the platform in sbt:

https://stackoverflow.com/questions/55591045/sbt-dynamically-detect-building-platform

**Comment by b-studios** [comment]
Hey, I incorporated your proposed fix in 203f717. Could you validate that it works for you now?

**Comment by b-studios** [comment]
The other commands (`generateLicenses`) are rarely used, so I close this ticket for now. But thanks for bringing it to my attention :)
